src/contrib/torch/tvm_module_wrapper/RuntimeModuleWrapperTorch.cc

```diff
@@ -0,0 +1,159 @@
+  }
+
+  if (src.dtype().isScalarType(torch::kBool)) {
+    auto temp = src.toType(torch::kUInt8);
+    return {.dl_managed_tensor = at::toDLPack(temp), .is_bool = true};
+  }
+
+  return {.dl_managed_tensor = at::toDLPack(src), .is_bool = false};
+}
+
+at::Tensor fromDLPackExt(const DLPackTensorExt& src) {
+  if (src.is_bool) {
+    return at::fromDLPack(src.dl_managed_tensor).toType(torch::kBool);
+  } else {
+    return at::fromDLPack(src.dl_managed_tensor);
+  }
+}
+
+/**
+ * @brief A Torch's module which wraps TVM's OperatorModule Class.
+ * The basic forward function calling TVM's runtime is provided.
+ * The TVM module can be serialized/deserialized as a Torch module.
+ */
+class OperatorModuleWrapper : public torch::jit::CustomClassHolder {
+ public:
+  OperatorModuleWrapper() { runtime_module = tvm_contrib_torch_get_last_saved_runtime_module(); }
+
+  void forward(const c10::List<at::Tensor>& inputs) {
+    int input_length = inputs.size();
+
+    std::vector<DLPackTensorExt> tensors;
+
+    for (int i = 0; i < input_length; ++i) tensors.push_back(toDLPackExt(inputs[i]));
+    tvm_contrib_torch_operator_module_forward(
+        this->runtime_module, static_cast<DLPackTensorExt*>(tensors.data()), tensors.size());
+
+    for (int k = 0; k < input_length; ++k) {
+      tensors[k].dl_managed_tensor->deleter(tensors[k].dl_managed_tensor);
+    }
+  }
+
+  std::string Serialize() { return std::string(tvm_contrib_torch_encode(runtime_module)); }
+
+  explicit OperatorModuleWrapper(std::string state) {
+    runtime_module = tvm_contrib_torch_decode(state.c_str());
+  }
+
+ private:
+  TVMContribTorchRuntimeModule* runtime_module;
+};
+
+/**
+ * @brief A Torch's module which wraps TVM's GraphExecutorFactory Class.
+ * The basic forward function calling TVM's runtime is provided.
+ * The TVM module can be serialized/deserialized as a Torch module.
+ */
+class GraphExecutorFactoryWrapper : public torch::jit::CustomClassHolder {
+ public:
+  explicit GraphExecutorFactoryWrapper(TVMContribTorchRuntimeModule* executor_factory)
+      : executor_factory_(executor_factory) {}
+
+  GraphExecutorFactoryWrapper()
+      : GraphExecutorFactoryWrapper(tvm_contrib_torch_get_last_saved_runtime_module()) {}
+  std::string Serialize() { return tvm_contrib_torch_encode(executor_factory_); }
+
+  explicit GraphExecutorFactoryWrapper(std::string state) {
+    executor_factory_ = tvm_contrib_torch_decode(state.c_str());
+  }
+
+  c10::List<at::Tensor> forward(const c10::List<at::Tensor>& inputs) {
+    int input_length = inputs.size();
+
+    TORCH_CHECK(input_length > 0, "Receive empty list of input tensors");
+
+    std::vector<DLPackTensorExt> tensors;
+
+    for (int i = 0; i < input_length; ++i) tensors.push_back(toDLPackExt(inputs[i]));
+
+    auto outputs = new DLPackTensorExt*;
+
+    auto num_outputs = tvm_contrib_torch_graph_executor_module_forward(
+        executor_factory_, static_cast<DLPackTensorExt*>(tensors.data()), tensors.size(), outputs);
```

Comment:
```suggestion
    DLPackTensorExt* outputs;

    auto num_outputs = tvm_contrib_torch_graph_executor_module_forward(
        executor_factory_, static_cast<DLPackTensorExt*>(tensors.data()), tensors.size(), &outputs);
```

It doesn't really need to new a pointer to `DLPackTensorExt*`. A pointer to the local variable (on the stack frame) will be easier to work with. And it doesn't need to `delete outputs` later on (but it still need to delete the DLPackTensorExt array allocated inside `tvm_contrib_torch_graph_executor_module_forward`)